ocp_resources/resource.py

```diff
@@ -200,7 +199,7 @@ def _cmp(self, other):
             return 1
 
 
-class classproperty(object):  # noqa: N801
+class ClassProperty:  # noqa: N801
```

Comment:
The `noqa` comment can be removed from this line now that this has been addressed.